### ISSUE TITLE
--realign-epochs options in nnet2 training scripts

### ISSUE THREAD

**Issue by eliemichel** [kaldi-asr/kaldi#191]
In help message:

```
--realign-epochs <list-of-epochs|"">           # A list of space-separated epoch indices the beginning of which
```

But:

```
train_pnorm_accel2.sh: invalid option --realign-epochs
```

There is a `realign_times` variable defined, but no `realign_epochs`, and nothing in the code to handle it anyway. Did I miss something?

(By the way, does this usually pays off?)

PS: I bet that this is the help message that is wrong, not the overall code.

**Comment by danpovey** [comment]
Thanks- I pushed a fix for the help message.
You could set it to a value like "0.3 0.6" or something like that.
We haven't done many experiments with realignment - I think we added support for it for some specific reason but I guess it didn't make much difference or we would have looked into it further.
However, Karel got some improvement from it, I think, in his setup.  It would be good if you could experiment with it a little, and let us know your experiences.

**Comment by vesis84** [comment]
Yes, I was getting small improvement of WER 0.2% from re-initializing of last layer and training again from initial learning rate (the difference between GMM and DNN alingments). It was a little but quite consistent gain...
